src/main/java/io/strimzi/kafka/metrics/KafkaMetricsCollector.java

```diff
@@ -59,15 +57,14 @@ public List<MetricFamilySamples> collect() {
             KafkaMetric kafkaMetric = entry.getValue();
             LOG.trace("Collecting Kafka metric {}", metricName);
 
-            String name = metricName(metricName);
+            String prometheusMetricName = metricName(metricName);
             // TODO Filtering should take labels into account
-            if (!config.isAllowed(name)) {
-                LOG.info("Kafka metric {} is not allowed", name);
+            if (!config.isAllowed(prometheusMetricName)) {
+                LOG.info("Kafka metric {} is not allowed", prometheusMetricName);
                 continue;
             }
-            LOG.info("Kafka metric {} is allowed", name);
-            LOG.info("labels " + metricName.tags());
-            MetricFamilySamples sample = convert(name, metricName.description(), kafkaMetric, metricName.tags());
+            LOG.info("Kafka metric {} is allowed", prometheusMetricName);
```

Comment:
I know this was already here. But do we really need this on INFO level? Maybe DEBUG would be enough?

Reply:
Yes we plan to address this in https://github.com/strimzi/metrics-reporter/issues/6

Reply:
It is changed in this PR and people have to review this, so it can be easily addressed right now I guess?